tests/unit/components/video-box/VideoBox.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import { describe } from 'vitest'
+import { screen, render } from '@testing-library/react'
+import VideoBox from '~/components/video-box/VideoBox.jsx'
+
+describe('VideoBox component test', () => {
+  beforeEach(() => {
+    render(<VideoBox />)
```

Comment:
I guess you can improve your test a bit by adding a video path test to make sure it's the right video. For this, you need to pass the corresponding property to the component and then check if the video path matches the one passed.
```javascript
render(<VideoBox video={videoSrc} />)
```